flasher/overlays/installer/usr/local/bin/installer.sh

```diff
@@ -136,7 +139,7 @@ if [ -z "${BOOTFS_PART}" ]; then
   mount PARTLABEL=results ${FLASHER_STORAGE_RESULTS}
   sfdisk -d ${CHOICE}
   # dump parititon table sector(i.e first) for debugging purpose.
-  hd ${CHOICE} -n 512 -s 0x0 > ${FLASHER_STORAGE_RESULTS}/parition_dump
+  hd ${CHOICE} -n 512MB -s 0x0 > ${FLASHER_STORAGE_RESULTS}/parition_dump
```

Comment:
512 bytes to 512 MB is a big jump and even for debugging attaching an 512MB file can be challenging. Why are we bumping it up so high?

Reply:
This was recommendation from @obbardc,  Mostly because this would have all the required information.

From last logging, the target disk behaviour appeared to be strange, so the more dump we have the better it is.
I can reduce the size and but not sure if we would miss the data. 
Let me know your choice here

Reply:
Maybe we can be a bit selective with it? The partition tables are either at the head or end of the device. What other information are we looking for?

Reply:
hmm.. let's go with 512KB ?

Reply:
My hope here was that we could see what was originally on the device and what has been written to the device (and compare manually). The first MB or so would be fine to capture I think.

Reply:
Okay, let's reduce the dump size to 512k and add comment that bmaptool is not writing the MBR and try again.

Reply:
Just pushed with above changes.